src/Command/CommandBase.php

```diff
@@ -1055,6 +1066,72 @@ protected function deleteSshKeyFromCloud(\stdClass $cloud_key): void {
     }
   }
 
+  /**
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   */
+  protected function checkForNewVersion(InputInterface $input, OutputInterface $output): void {
+    try {
+      if (strpos($input->getArgument('command'), 'api:') === FALSE && $this->hasUpdate()) {
```

Comment:
Why don’t we check for an update when running API commands?

Reply:
I have no idea, this is how you coded the original update command 😄 https://github.com/acquia/cli/pull/284/commits/2a38befb7f4c27e4ce57d139b1e9f2d6211fee59

Reply:
I did think it might have been me. Oh wait! I think it's so that we don't break the JSON output of api:* commands. Let's add a comment to that effect so that we don't forget.